docs/admin/pool_operations.md

```diff
@@ -453,7 +462,7 @@ $ dmg pool exclude --rank=${rank} --target-idx=${idx1},${idx2},${idx3} <pool_lab
 
 The pool target exclude command accepts 2 parameters:
 
-* The rank of the target(s) to be excluded.
+* The engine rank of the target(s) to be excluded.
```

Comment:
nit: should be "rank(s) of the target(s) to be excluded", "rank of the targets to be excluded" isn't correct because we don't identify a group of targets by a single rank

Reply:
is this really correct? AFAIK a single "pool exclude" command invocation can only exclude (one or more targets) from a single engine rank, not from multiple engines?

Reply:
apologies, you are correct, ignore this comment